catalog/core/catalog-core-solrcommands/pom.xml

```diff
@@ -68,6 +68,12 @@
         <dependency>
             <groupId>ddf.security.core</groupId>
             <artifactId>security-core-api</artifactId>
+            <exclusions>
+                <exclusion>
+                    <artifactId>log4j-api</artifactId>
+                    <groupId>org.apache.logging.log4j</groupId>
+                </exclusion>
+            </exclusions>
```

Comment:
An API bundle (security-core-api) that depends on log4j API. It feels like the security-core-api should depend on an SLF4J API because ultimately log4j is just one of several libraries that can be used for logging. This is not a comment on this PR, just an observation. What do you think?